docs/content/deployments/server/deploying-on-microtica.md

```diff
@@ -0,0 +1,112 @@
+
+### 3. Connect an AWS account 
+
+In the last step you select the environment in which you want to deploy the template. An existing default environment called `development` will be preselected here, or you can create a new environment.  
+
+Connect your AWS account when prompted. This process takes only a few seconds, so afterward only choose the region you want to deploy in. 
+
+![Connect AWS account](https://global-uploads.webflow.com/62b04b3a715c202b5fa1408b/639727f98144a7eb845dabee_GYZFu6wD5huAf6H5sCs4qpz09wQOaI2IMThsEd0CwR4PePc1ryF4mAlb4YkTZdIkX3Mnh_OzEr9bdm2r1_5GkYHHlFgHksP_hDj7DmlnitZmWaotj5th3R4wHK6gJzhIDmjfXeCk30YdK1kaeGH41JRCyteVH9y3YqzIyGz8xQlcDYKCoWVxDey47mPw.png)
+
+### 4. Deploy to AWS
+
+Finally, a deployment summary of what will be provisioned on your AWS account is presented. Click on **Deploy** to trigger a deployment of the template and start creating the infrastructure for a Medusa Server.
+
+![Deployment Summary](https://global-uploads.webflow.com/62b04b3a715c202b5fa1408b/639727f932d1b32052287126_h96ejVytOhLLEUCeIWbQu-OG4oxJQqj5z9ZJXjH5EboT5gnw6s61hl2mvHcUaGQUyJMQtNdyb4hdFWDvYlFBa8wznkBus7nkXR_0JAIRvN5QDdZP6WCDNSzYeBSJcoV-ifDdUHzJODF_Oxv6mgIsHtpj92PA6VgQsaYM0n_xyzV-lhalH6kdR-tku2hb.png)
+
+**It will take around ten minutes for the solution to be deployed on your cloud account.**
+
+You can follow the build pipeline in real-time by clicking the **View Logs** button. 
+
+![Building Resources](https://global-uploads.webflow.com/62b04b3a715c202b5fa1408b/639727f9a465c27cf7090e5f_gxwdXAAmUxHsSyJlgScHuu4m7GiFTNnwW-OZoAhDC8v-Ez3KmsK-OCKfF2Qe2JUtxMyih7jCA7n-hAZnx0fdFjv-TWyDuoAUvftSRE9hqmPB1gVhG6lH0IhXqbliou1VnAVDvlCtHobzlUrrDhnCmzob9PfdNGrzlhwsL60QsAXJm1gIPE27J6P_Y2Y3.png)
+
+Once the build process is complete, a new deployment with the infrastructure resources is triggered. You can follow the logs of the deployment process by clicking the **View deployment** button, and then selecting the deployment from the list. 
+
+![Deployment Logs](https://global-uploads.webflow.com/62b04b3a715c202b5fa1408b/639727f93aa3974438e56689_vF-4U1L5B_sQQJdrjaE_bdlN2hdTsOcn5eWVRfgzF1UqNAuWAqN2zh9ttWp6SW2jjCqPUNLpRsL4j01FR5emAMcJqCbfwbaBv3Np49JH6hpggYm3rmY-xYCOg8H0dooKnC4x_sTD-JreveLzUxBMxQI3M0gCm-8B2YJ5eJj7dU8mf8qlSTYmnpnExIvB.png)
+
+---
+
+## Preview the environment
+
+After the deployment is finished, navigate to *Resources → [AppName] (Medusa in our case) → Overview*, and under *Resource Outputs* you should see the **AccessUrl**. This is the server url that you can use to access API endpoints. Try getting the list of products using the endpoint `/store/products`.
+
+![Preview Medusa Server](https://global-uploads.webflow.com/62b04b3a715c202b5fa1408b/639727f9d423043b2f685a93_MsVHdJXNiV3oswGWvMAwZRG-ybWwm6QdwPynjQFNr6aFrKMCuG0oQZjxLkzct_dqkM6mLOdqi9SQJwiv_ZQJCPUNr15zTDBah3GFtT8r6EVlgAOS9ylJ0ObAVAfYMSuiXre25qz_4GywqeQT-8Ieh5CwYBTd_Y26jvZgUo75KbU8ftzFLjRoRxmU3GNu.png)
```

Comment:
I think it's not necessary to include the image of the output. Maybe instead an image showcasing where to find the URL.